### DISCUSSION TITLE
add url to packages in wolfi

### DISCUSSION THREAD

**Discussion by westonsteimel** [wolfi-dev/os#652] [Ideas]
I was wondering if there is any plan to add project url information to the Wolfi packages similar to what exists for alpine packages?  An example would be for [make](https://pkgs.alpinelinux.org/package/edge/main/x86_64/make) where the project url is https://www.gnu.org/software/make and that then gets persisted into the apk metadata.

I'm happy to start adding these in for existing packages if that's something that would be desirable.  From what I could tell from looking at the melange code it appears to already be supported but just isn't yet being populated in any of the yaml files

**Comment by kaniini** [comment]
Yeah, we can do it.
